scripts/pi-hole/js/db_queries.js

```diff
@@ -273,7 +273,7 @@ $(document).ready(function() {
     });
     $("#all-queries tbody").on( "click", "button", function () {
         var data = tableApi.row( $(this).parents("tr") ).data();
-        if (data[4] === "1")
+        if (data[4] === 1 || data[4] === 4 || data[5] === 5)
```

Comment:
I cannot test this right now, but you change from comparing to a string to comparing to a number. This is important sue to the `===` operator. Is that change intended? If so, did it actually work *at all* before your change?

Reply:
This change is intended, and it was broken previously.
As seen in the `switch(data[4])` block, `data[4]` is expected to be a number instead of a string.